frontend/src/utilities/cleanup_text_input.ts

```diff
@@ -0,0 +1,5 @@
+// Necessary because innerText puts an extra newline character at the end when the text is more than one line.
```

Comment:
Pease use `kebab-case` for TypeScript files per the existing convention. Although you can probably avoid using a single-purpose file in `/utilities` by just exporting it from `input.ts` since it's such a short function and is already used in `input.ts` (in addition to `Document.vue`).

Reply:
I committed the change so we could get this merged while you're asleep.